astro/src/content/blog/digital-identity.mdx

```diff
@@ -0,0 +1,55 @@
+---
+publish_date: 2024-02-20
+title: Digital Identity - Human Elements of a Connected World
+description: Your identity is so much more than the pieces of paper and plastic that sit in a wallet. In today's world, your digital identity is valuable, and ever-changing.
+authors: Brad McCarty
+image: /img/blogs/digital-identity/digital-identity.png
+categories: Education
+tags: digital identity
+excerpt_separator: "{/* more */}"
+---
+Every time you key in credentials to access your go-to app, sift through your inbox, scroll social media, or jump into an [online game](industries/gaming-entertainment), you're doing more than just inputting a username and password.{/* more */} You're affirming your digital identity, declaring, "Yep, that's me!" And here's the kicker: we're all engaging in this ritual almost daily, likely thousands of times each month. But we seldom pause to ponder the dance happening behind the screen. So let's unpack the layers around digital ID, the daily login routine, CIAM providers, and the world of authentication.
+
+## What is Digital Identity?
+
+Imagine identity as the digital equal to your DNA. It contains every choice, preference, and action you make online. It's your virtual version of all of your personal data, evolving with every click, every preference tweak, and every new interaction. This identity isn't static. It's a living, breathing entity in the digital world, reshaping itself as you navigate through your online life.
```

Comment:
I am not 100% comfortable with the metaphor to the DNA. Your ID is not you, it represents who you are. If I cut up my driver's license, nothing happens to me. Your IDs are representations of you, not you.  The ID itself does not contain the choices and actions.  It allows you to make those choices that represent you. It is a key to the virtual version of all your data.  I tend to think of identity's as rather static. What you do with identity changes.  This could be a personal thing and if you are comfortable with the analogy, it is a vivid picture and reads well.

Reply:
Changed the analogy. Let's see if that works better.